### ISSUE TITLE
Add authorization

### ISSUE THREAD

**Issue by lavgl** [roman01la/cljs-rum-realworld-example-app#6]
I'd like to implement auth as a next part. This covers:
- [x] 'Sign in' page
- [x]  'Sign up' page
- [x]  Differentiating navigation menu for logged and non-logged users
- [ ]  'Log out' button on 'Settings' page
- [x]  Storing user info and jwt token

I suppose this functionality will be covered by several PRs.

**Comment by roman01la** [comment]
LGTM

**Comment by foopang** [comment]
Sorry for jumping in, but I have some code ready for review if you don't mind me taking this task.

**Comment by lavgl** [comment]
Sure I don't. Do you want to cover all the functionality described above?

**Comment by foopang** [comment]
Yes, I have all the functionality described above covered already in https://github.com/roman01la/cljs-rum-realworld-example-app/pull/7. But the PR is not approved by @roman01la. I will try to break it down into several PRs.

**Comment by roman01la** [comment]
@foopang Thanks!

**Comment by roman01la** [comment]
@foopang One thing that might save you a bit of time: try to stick to a code style of the project as much as you can, for example component names should be camel cased starting with a capital character. Basically see how the rest of the code is written and try to write your code in same manner. We want the code base to be consistent, because this project is intended to serve as a reference for others.
